code/game/objects/structures/electricchair.dm

```diff
@@ -7,19 +7,19 @@
 	var/last_time = 1.0
 	var/delay_time = 50
 
-/obj/structure/chair/e_chair/New()
-	..()
+/obj/structure/chair/e_chair/Initialize(mapload)
+	. = ..()
 	overlays += image('icons/obj/chairs.dmi', src, "echair_over", MOB_LAYER + 1, dir)
-	spawn(2)
```

Comment:
I see no reason for this to be inside a `spawn()` bracket, but if anyone has insight feel free to inform

Reply:
Because `part` is not an argument and is passed later when creating it.
![image](https://user-images.githubusercontent.com/15887760/102718087-81a64180-42e6-11eb-8375-7c1b633a0ad7.png)

Reply:
TODO: Refactor that to take it as an arg

Reply:
Done